### ISSUE TITLE
Create a new Bearer Token 

### ISSUE THREAD

**Issue by jahnavisana2812** [supabase/supavisor#338]
Is there any method to create our own bearer token instead of using the default bearer token?

**Comment by J0** [comment]
Hey @jahnavisana2812 

Thanks for the query - quick check - are you self hosting? I haven't tried this but I think the token should be a regular JWT so if I'm not mistaken you should be able to create a new token and sign it with the [JWT secret set](https://github.com/supabase/supavisor/blob/72a4ab830d95a5ca39b764f0b9ae1fe5cac7d893/Makefile#L8) 

A tool like [jwt.io](https://jwt.io/) might help. 

Let me know if that works

**Comment by jahnavisana2812** [comment]
Yeah self-hosting on kubernetes with self-hosted postgres.So these are the default variables I found for the yaml.What all values can be changed?
          env:
            - name: SECRET_KEY_BASE
              value: '12345678901234567890121234567890123456789012345678903212345678901234567890123456789032123456789012345678901234567890323456789032'
            - name: VAULT_ENC_KEY
              value: '12345678901234567890123456789032'          
            - name: API_JWT_SECRET
              value: dev
            - name: METRICS_JWT_SECRET
              value: dev
            - name: REGION
              value: local
            - name: ERL_AFLAGS
              value: -proto_dist inet_tcp

**Comment by J0** [comment]
I don't quite have the time to try atm (earliest would be weekend) but I think you can change `ENC_KEY`, `API_JWT_SECRET`, and `METRICS_JWT_SECRET` so long as you use bearer tokens which are appropriately signed.

If it helps, you could possibly paste the errors here and I'm happy ot take a look when a slot frees up or perhaps team might jump in when free

**Comment by abc3** [comment]
@jahnavisana2812 supavisor receives metrics and api jwt tokens as Bearer keys. You can use whichever one you prefer
